info/guide/guide-chapter-3.md

```diff
@@ -366,6 +366,19 @@ Note that all comparison operators, such as `==`, `>`, `<`, should not be split.
 ```kotlin
 if (condition) list.map { foo(it) }.filter { bar(it) } else list.drop(1)
 ```  
+
+**Note:** If dot qualified expression is inside condition or passed as an argument - replace with new variable
+
+**Invalid example**: 
+```kotlin
+     if (node.text.length.dec() != 0) {}
+```
+ 
+**Valid example**: 
+```kotlin
+        val nodeLen = node.text.length.dec()
```

Comment:
This is not a very good variable name to be used in an example :) It's not node length, it's node's *text* length minus one. Please either find a more straightforward example or rename the variable to be more self-descriptive.